clang/tools/driver/driver.cpp

```diff
@@ -235,6 +236,9 @@ int clang_main(int Argc, char **Argv, const llvm::ToolContext &ToolContext) {
   if (llvm::sys::Process::FixupStandardFileDescriptors())
     return 1;
 
+  // clang-format off
+  // Cratels:初始化所有的target.如果在cmake configure的时候指定了target,这只会初始化对应的target,否则就初始化默认的targets.
```

Comment:
I prefer add this comment in cmake scripts (but I never read cmake scripts and am not sure if this is added).